src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/render/ModuleRotations.kt

```diff
@@ -41,11 +42,15 @@ import net.minecraft.util.Pair
 
 object ModuleRotations : Module("Rotations", Category.RENDER) {
 
-    val showRotationVector by boolean("ShowRotationVector", false)
+    private val showRotationVector by boolean("ShowRotationVector", false)
+    private val smoothRotations by boolean("SmoothRotations", false)
+    private val smoothingFactor by float("SmoothFactor", 0.15f, 0.1f..0.9f)
```

Comment:
Then make sure this is in a toggleable configurable at least, so it's "Smooth Rotation" -> "Factor". Make sure it's "Rotation", not "Rotations", so it matches the "ShowRotationVector"

Reply:
I'll work on it when I get on my pc